practice/case.py

```diff
@@ -0,0 +1,59 @@
+"""
+Prompts user for a ward in snake_case or CamelCase and converts it into the opposite case.
+"""
+
+# setup: import regex and get word from user
+import re
+
+
+def get_user_word():
+    """Asks user for the word to be converted.
+    """
+    word_to_change = input('Please give me a word and I will convert it to snake_case or CamelCase. ')
+    return word_to_change
+
+
+def check_word_case(word_to_change):
+    """Checks if word_to_change needs to be converted to snake or camel case.
+    >>> check_word_case('snake_case')
+    'SnakeCase'
+     """
+    if word_to_change.islower() == True: # convert to camel
+        final_output = convert_to_camel(word_to_change)
+    if word_to_change.islower() != True: # convert to snake
+        final_output = convert_to_snake(word_to_change)
```

Comment:
These if statements are disjoint. You don't want both to ever run. In that case use if-else.